transforms/__testfixtures__/update-react-imports/react-already-used-named-export.input.js

```diff
@@ -0,0 +1,3 @@
+import * as React from 'react';
```

Comment:
is the .output.js for this file meant to be empty? (I can't leave this comment on the output file because it's empty)

Reply:
Yeah. Empty output means the file won't be edited (we return null).